stream.go

```diff
@@ -160,12 +158,18 @@ func (s *Stream) Read(max int) (events []Event, err error) {
 	buffer := make([]C.PmEvent, max)
 	numEvents := C.Pm_Read(unsafe.Pointer(s.pmStream), &buffer[0], C.int32_t(max))
 	events = make([]Event, numEvents)
+
 	for i := 0; i < int(numEvents); i++ {
+
+		buf := new(bytes.Buffer)
```

Comment:
I tried experimenting with `copy()`, but `buffer[i].message` isn't a string or slice and the endian-ness need to be flipped. It got messy really fast. I am going to experiment with it more, and look at the specs some more to confirm I am using everything correctly. If you can nudge me more in the right direction I'd appreciate it. Thanks!